README.md

```diff
@@ -1 +1,16 @@
-# CAPS
\ No newline at end of file
+# CAPS
+
+# Challenge
+- CAPS Phase 1: Begin the build of an application for a company called CAPS - The Code Academy Parcel Service. In this sprint, we’ll build out a system that emulates a real world supply chain. CAPS will simulate a delivery service where vendors (such a flower shops) will ship products using our delivery service and when our drivers deliver them, be notified that their customers received what they purchased.
+
+- Tactically, this will be an event driven application that “distributes” the responsibility for logging to separate modules, using only events to trigger logging based on activity.
+
+# Approach & Efficiency
+
+
+# API 
+
+
+# Solution
+<!-- embedded whiteboard image -->
+![whiteBoard]( )
```

Comment:
Not the right readme templete(this is for Code Challenges) you need the [ASSIGNMENT_TEMPLATE.md](https://github.com/codefellows/seattle-javascript-401n18/blob/main/ASSIGNMENT_TEMPLATE.md) that is in the class repo